scrapy/extensions/feedexport.py

```diff
@@ -131,18 +132,28 @@ def __init__(self, uri, access_key=None, secret_key=None):
     @classmethod
     def from_crawler(cls, crawler, uri):
         return cls(uri, crawler.settings['AWS_ACCESS_KEY_ID'],
-                   crawler.settings['AWS_SECRET_ACCESS_KEY'])
+                   crawler.settings['AWS_SECRET_ACCESS_KEY'],
+                   crawler.settings['FEED_STORAGE_S3_ACL'])
 
     def _store_in_thread(self, file):
         file.seek(0)
         if self.is_botocore:
+            kwargs = dict()
+            if self.acl:
+                kwargs.update(dict(ACL=self.acl))
```

Comment:
dict() is not necessary here, as .update supports keyword arguments. Also, as it is a single value, it can just an assignement: `kwargs['ACL'] = self.acl`, or `kwargs = dict(ALC=self.acl)`. If you go with kwargs=.. assignement, it can be made shorter further, as `kwargs = {'ACL': self.acl} if self.acl else {}`. 

Whether to use if/else assignement is up to stylistic preferences, but I think that'd be good to at least replace `kwargs.update(dict(ACL=self.acl))` with `kwargs['ACL'] = self.acl`.

Reply:
Done.